build_tools/cmake/iree_run_module_test.cmake

```diff
@@ -215,9 +213,8 @@ endfunction()
 #
 # Parameters:
 #   NAME: Name of the target
-#   BENCHMARK_MODULE_SRC: IREE module flagfile path built from benchmark_suite.
-#       The flagfile for different compile configurations are stored in the
-#       subdirectories.
+#   MODEL: Model "<UUID>_<name>" defined in
+#       "build_tools/python/e2e_test_framework/models".
```

Comment:
I can't find UUID information in https://github.com/iree-org/iree/blob/main/build_tools/python/e2e_test_framework/models. Could we be more specific?

Reply:
Added some more description.